src/main/java/uk/gov/companieshouse/mapper/email/EmailMapper.java

```diff
@@ -13,9 +13,9 @@
 import static uk.gov.companieshouse.model.Constants.EMAIL_TOPIC;
 
 @Service
-public class EmailMapper<T> {
```

Comment:
Removed generic from class and move to a method level generic instead. Otherwise EmailMapper could not be reused for multiple types of email in DissolutionEmailService cc @matthewdoreilly @faggc

Reply:
👍  I have that fix in my branch as well. Noticed we weren't using the generics properly first time